### ISSUE TITLE
Cannot remove/uninstall

### ISSUE THREAD

**Issue by kklot** [ozmartian/vidcutter#320]
on Mac OS, installed the latest release, installed, then deleted the app from `Application`  but the icon still shows up on Mac.

Moreover, while this app's icon had been moved into a group before, now it shows (in addition to the one inside the group) outside the group – all while it has been deleted from `Application`.

(the app cannot start by the way)

**Comment by ozmartian** [comment]
Are you on Big Sur?

Try this re removing the icon.

```
Click “Launchpad” in the Dock > Find the app icon that you want to remove > Press and hold the Alt (⌥) key, or click and hold the app icon until it jiggles > Click the cross next to the app that you want to remove and click “Delete” to remove the app and app icon immediately form your Mac.
```

**Comment by ozmartian** [comment]
What version of macOS are you on? Big Sur?

**Comment by kklot** [comment]
Yes, this is the current state, removing it it will reappear later. How to completely remove it?

![Screenshot 2021-08-18 at 12 10 43](https://user-images.githubusercontent.com/4459378/129880860-531b092f-8f0b-4185-8b21-79c4fee3c804.png)
